packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -0,0 +1,108 @@
+import { ID, SquareSizes } from '@audius/common/models'
+import { cacheCollectionsSelectors } from '@audius/common/store'
+import {
+  Divider,
+  Flex,
+  // IconHeart,
+  // IconRepost,
```

Comment:
👀

Reply:
ah, yes we may want to chat about this. I'm learning there are some limitations to pre-building the icons in harmony... essentially title prop doesn't seem to work, because that needs to be compiled real time

Reply:
Im a little confused, do these commented imports achieve some sort of pre-building thing?

Reply:
no sorry for confusion, i should remove them, they are imported below from /harmony/src/...